### ISSUE TITLE
Module not found: Can't resolve 'react-regl' 

### ISSUE THREAD

**Issue by Nizarius** [kevzettler/react-regl#10]
Steps to reproduce:
```
npx create-react-app new_app
cd new_app
npm install --save react-regl
```

Then use this code in src/App.js:

```
import React from "react";
import ReactDOM from "react-dom";
import Regl from "react-regl";

export default class App extends React.Component {
  constructor(props) {
    super(props);
  }

  componentDidCatch(e, info) {
    console.log(e, info);
  }

  render() {
    return <Regl width={500} height={500} color={[0.5, 0.5, 0.5, 1]} />;
  }
}
```

Then:
`npm run start`

What I do wrong?

**Comment by hydrosquall** [comment]
It seems that you might have to change your react/react-dom versions from 16.6 or whatever the latest is (since create-react-app currently installs the latest 16.8), down to 16.1. See #9 for more details about the error that gets encountered.

**Comment by kevzettler** [comment]
fixed in #11
